Sources/ProjectDescription/DeploymentTarget.swift

```diff
@@ -5,7 +5,7 @@ import Foundation
 /// A supported minimum deployment target.
 public enum DeploymentTarget: Codable, Hashable {
     /// The minimum iOS version and the list of devices your product will support.
-    case iOS(targetVersion: String, devices: DeploymentDevice)
+    case iOS(targetVersion: String, devices: DeploymentDevice, supportsMacDesignedForIPhone: Bool = true)
```

Comment:
I would call this ForIOS because it's iPhone and/or iPad
```suggestion
    case iOS(targetVersion: String, devices: DeploymentDevice, supportsMacDesignedForIOS: Bool = true)
```